variables.tf

```diff
@@ -59,6 +84,7 @@ variable "bastion" {
 }
 
 variable "bootstrap" {
+    type    = object({ count = number, memory = string, processors = string })
```

Comment:
@Elayaraja-Dhanapal last one.. can you please add a validation for count  == 1 .

Reply:
It is 0 or 1.
I have this validation already in place.

Reply:
ah ok.. missed it